Main.java

```diff
@@ -0,0 +1,50 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Main {
+    static String scanLine() {
+        Scanner scan = new Scanner(System.in);
+        return scan.nextLine();
+    }
+    public static void main(String[] args) {
+        int start, end, length, var, aux;
+        int mean = 0;
+        System.out.println("Arithmetic mean algorithm");
+        System.out.println("""
+            Write 1 or 2 depending the case:\s
+            1 - arithmetic mean of a certain number range\s
+            2 - arithmetic mean of a certain count of numbers""");
+        var = Integer.parseInt(scanLine());
+        switch (var) {
+            case 1 -> {
+                System.out.println("Write the initial start of the range");
+                start = Integer.parseInt(scanLine());
+                System.out.println("Write the end of the range");
+                end = Integer.parseInt(scanLine());
+                if(start > end){
+                    aux = start;
+                    start = end;
+                    end = aux;
+                }
+                for(int i = start; i <= end; ++i) {

```

Comment:
Exista o metoda matematica ce poate face treaba asta, fara a itera. Look it up 😄